test/membrane_rtmp_plugin/rtmp_sink_test.exs

```diff
@@ -0,0 +1,112 @@
+defmodule Membrane.RTMP.Sink.Test do
```

Comment:
A test for file -> RTMP sink -> RTMP source -> file would be useful. I'd consider making it an example too.

Reply:
As discussed today on meeting with @daniel-jodlos and @bblaszkow06 I will attempt to implement such an integration test. However in case of difficulties its implementation will be moved to a separate issue.

Reply:
Cool, it can be a separate issue